### PR TITLE
Update browser.html.twig

### PR DESCRIPTION
**Pull request by lucasgranberg**
| Q             | A
| ------------- | ---
| Fixed tickets | -
| License       | MIT

* add paginator class
* remove unused columns
* modify how image formats are displayed

### CODE REVIEW

File: Resources/views/Ckeditor/browser.html.twig [hunk 1]
```diff
@@ -76,28 +76,14 @@ file that was distributed with this source code.
...
-                                    {% if sortable %}<a href="{{ admin.generateUrl('ckeditor_browser', sort_parameters) }}">{% endif %}
-                                        {{ admin.trans(field_description.label) }}
-                                        {% if sortable %}</a>{% endif %}
-                                </th>
-                            {% endspaceless %}
-                        {% endif %}
-                    {% endfor %}
+                    {% spaceless %}
+                        <th class="sonata-ba-list-field-header-image">
+                            Image
```
Comment (OskarStark) [inline Resources/views/Ckeditor/browser.html.twig]: can you please use translations keys here?

File: Resources/views/Ckeditor/browser.html.twig [hunk 2]
```diff
@@ -76,28 +76,14 @@ file that was distributed with this source code.
...
+                        </th>
+                        <th class="sonata-ba-list-field-header-formats">
+                            Formats
```
Comment (OskarStark) [inline Resources/views/Ckeditor/browser.html.twig]: can you please use translations keys here?

Comment (lucasgranberg) [inline Resources/views/Ckeditor/browser.html.twig]: This PR is over 1 year old. Is it even viable to try to merge it?

Comment (Soullivaneuh) [inline Resources/views/Ckeditor/browser.html.twig]: @lucasgranberg Don't know. Just see the actual system and see by yourself. :+1:

Comment (Soullivaneuh) [inline Resources/views/Ckeditor/browser.html.twig]: Otherwise, please close this PR.

**Pull request closed without merging**